### DISCUSSION TITLE
1000th person to star

### DISCUSSION THREAD

**Discussion by Lucas256-cmd** [st3v3nmw/obsidian-spaced-repetition#743] [General]
Let's goo. I'm the 1000th person to star this repo. I'm using this plugin for a few days now and I really like it. Especially because it runs fully in obsidian and i don't have to open an app like Anki. Tomorrow I have a test and I made flashcards about it, I hope it helped. Also I have an idea for a new feature. I would find it useful if I had a button to review my flashcards whenever I want. Right now I have to wait a day or two until I can review again. Or maybe I haven't found this feature but is it already in the plugin. Please tell me if there is a convenient way to do this.

**Comment by tosbourn** [comment]
You can use the cram feature to do this. Type "cram" into the command palette to start.
